keras/engine/input_layer.py

```diff
@@ -107,9 +115,32 @@ def get_config(self):
         config = {'batch_input_shape': self.batch_input_shape,
                   'dtype': self.dtype,
                   'sparse': self.sparse,
-                  'name': self.name}
+                  'name': self.name,
+                  'is_placeholder': self.is_placeholder}
```

Comment:
This specific change is problematic because it affects saved models and the loading contract. This is not a change confined to multi-backend Keras, since other Keras implementations rely on the same saved files: tf.keras, TF.js, etc.